modules/installation-user-infra-machines-advanced.adoc

```diff
@@ -0,0 +1,338 @@
+was before (the same offset), and the filesystem will be intact.
+
+For a PXE installation:
+
+This APPEND option preserves the sixth partition:
+
+[source,terminal]
+----
+coreos.inst.save_part=6
+----
+
+This APPEND option preserves the all partitions from 5 and higher:
+
+[source,terminal]
+----
+coreos.inst.save_part=5-
+----
+
+This APPEND option preserves any partition in which the partition label begins with `data` (`data*`):
+
+[source,terminal]
+----
+coreos.inst.save_partindex=’data*’
+----
+
+[id="installation-user-infra-machines-advanced_ignition_{context}"]
+== Identifying Ignition configs
+There are different ways to manage Ignition config files when you do
+bare metal installations. To begin with, there are two different types
+of Ignition configs you can provide during {op-system-first} installation and two
+different reasons for providing them:
+
+* **Live install Ignition config**: This Ignition config should be rarely
+used and is one you create manually from scratch. It is passed to the live
+install medium and it is run immediately upon booting that live medium to do setup
+tasks BEFORE the process that actually installs the {op-system-first} system to disk.
+This should only be used for performing tasks that need to be done once and
+not applied again later, such as some advanced partitioning that cannot be done using MachineConfigs. There are currently no officially supported procedures for using Ignition configs in this way.
++
+For PXE or ISO boots, you can create the Ignition config
+and APPEND the `ignition.config.url=` option to identify the location of
+the Ignition config.
+
+* **Permanent install Ignition config**: Every bare metal {op-system-first} installation
+needs to pass one of the Ignition config files generated by `openshift-installer`,
+including `bootstrap.ign`, `master.ign` and `worker.ign`, to the carry out the
+installation. It is not recommended to modify these files.
++
+For PXE installs, you pass the Ignition configs on the APPEND line using the
+`coreos.inst.ignition_url=` option. For ISO installs, after the ISO boots to
+the shell prompt, you identify the Ignition config on the `coreos-installer`
+command line with the `--ignition-url=` option.
++
+Instead of passing the location of an Ignition config via a kernel or
+command-line option, you can embed an Ignition config into the ISO
+installer image. This allows you to do bare metal installs with the ISO,
+without requiring access to an HTTP server. See “Embedding an Ignition
+config in the {op-system-first} ISO” for details.
+
+[id="installation-user-infra-machines-advanced_embedignition_{context}"]
+=== Embedding an Ignition config in the {op-system-first} ISO
+The following procedures describe how to embed an Ignition config into
+the ISO so it is applied when the new installation first boots from disk.
+
+To embed an Ignition config named `worker.ign` into an ISO image
+(for example rhcos-<version>-live.x86_64.iso), copy the image to
```

Comment:
Per ISG, use commas rather than parentheses to offset the text here:

> To embed an Ignition config named `worker.ign` into an ISO image, for example `rhcos-<version>-live.x86_64.iso`,